django_app/frontend/src/styles.scss

```diff
@@ -170,82 +170,167 @@ body {
-  fill: govuk-colour("green");
+.feedback__container--2.feedback__container--negative img {
+  transform: scaleY(-1);
 }
 
-/* Only show feedback when correct buttons pressed */
-.rb-response-feedback__thumbs-up,
-.rb-response-feedback__thumbs-down {
+/* Only show feedback buttons on last message, for now */
+feedback-buttons {
   display: none;
 }
-.rb-response-feedback__buttons[data-status="up"]
-  ~ .rb-response-feedback__thumbs-up {
-  display: block;
-}
-.rb-response-feedback__buttons[data-status="down"]
-  ~ .rb-response-feedback__thumbs-down {
+feedback-buttons:last-of-type {
   display: block;
 }
 
-.rb-streaming-response-loading {
+/* Response feedback - general styles */
+.feedback__container {
+  font-size: 0.75rem;
+  margin-bottom: 1.5rem;
+  margin-top: -0.25rem;
+  padding: 0 1rem;
+}
+.feedback__container--1:not([hidden]) {
+  align-items: flex-start;
+  color: #6b7280;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 0.625rem;
+}
+.feedback__star-help-text {
+  max-width: 3rem;
+  text-align: center;
+}
+.feedback__heading {
+  color: black;
+  font-size: 0.75rem;
+  font-weight: 400;
+  margin: 0;
+  margin-right: 1rem;
+}
+.feedback__star-container {
   align-items: center;
   display: flex;
-  padding: 0.5rem 0;
-  gap: 0.25rem;
 }
-.rb-response-loading {
+.feedback__star-button {
+  background-color: transparent;
+  border: 0;
+  color: white;
+  cursor: pointer;
+  padding: 0 0.25rem;
+}
+.feedback__star-button--hover {
+  color: #bebebe;
+}
+
+.feedback__response-container {
   align-items: center;
   display: flex;
-  padding: 0.5rem 0;
-  visibility: hidden;
+  gap: 0.5rem;
 }
-.rb-response-loading:focus-visible {
-  outline: 1px solid;
+.feedback__improve-response-btn,
+.feedback__rate-again-btn {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  display: block;
+  margin-bottom: 1.25rem;
+  margin-top: 0.5rem;
+  padding: 0;
+  text-decoration: underline;
+  text-underline-offset: 4px;
 }
-.rb-response-loading__spinner,
-.rb-streaming-response-loading__spinner {
-  width: 1.5rem;
+
+.feedback__chips-container {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+.feedback__chips-legend {
+  padding: 0;
+}
+.feedback__chips-inner-container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0.75rem 0;
+}
+.feedback__chip {
+  height: 0.75rem;
+  margin: 0;
+  opacity: 0;
+  width: 0.75rem;
+}
+.feedback__chip-label {
+  border: 1px solid #b1b4b6;
+  border-radius: 0.25rem;
+  color: #60697b;
+  cursor: pointer;
+  display: flex;
+  gap: 0.25rem;
+  left: -0.75rem;
+  margin-top: 0.5rem;
+  padding: 0.5rem 0.75rem;
+  position: relative;
+}
+.feedback__chip:focus-visible + .feedback__chip-label {
+  outline: 2px solid;
+}
+.feedback__chip:checked + .feedback__chip-label {
+  background-color: #b1b4b6;
+  color: black;
+}
+.feedback__chip-label img {
+  display: none;
+  width: 0.75rem;
+}
+.feedback__chip:checked + .feedback__chip-label img {
+  display: block;
+}
+.feedback__text-input {
```

Comment:
Should this be `resize: none;`? The resizing handle makes it look notably different from the other textareas. 
<img width="539" alt="Screenshot 2024-07-11 at 12 55 07" src="https://github.com/i-dot-ai/redbox/assets/23265724/743fcb83-8a2a-47b9-aace-714de6031d8c">

Reply:
@KevinEtchells ?

Reply:
As it auto-resizes, that sounds like a good idea